### PR TITLE
use build full name to create appVersionTag

### PR DESCRIPTION
**Pull request by serge17**
If the job is located inside a folder, the resulting appVersionTag will be "applicationId-folderPath/buildName-buildNumber". If it is not in a folder: "applicationId-buildName-buildNumber"

### CODE REVIEW

File: src/main/java/com/aspectsecurity/contrast/contrastjenkins/VulnerabilityTrendHelper.java [hunk 1]
```diff
@@ -249,17 +249,12 @@ public static AgentType getAgentTypeFromString(String type) {
         }
     }
 
-    public static String buildAppVersionTag(Run<?, ?> build) {
```
Comment (gmoore) [inline src/main/java/com/aspectsecurity/contrast/contrastjenkins/VulnerabilityTrendHelper.java:252]: Why did this method go away?

File: src/main/java/com/aspectsecurity/contrast/contrastjenkins/VulnerabilityTrendStep.java [hunk 2]
```diff
@@ -178,6 +179,9 @@ public Step newInstance(Map<String, Object> arguments) {
             if (arguments.containsKey("queryBy")) {
                 Object queryBy = arguments.get("queryBy");
 
+                step.setQueryBy((int) queryBy);
+            } else if (arguments.containsKey("appVersionTagFormat")) {
```
Comment (gmoore) [inline src/main/java/com/aspectsecurity/contrast/contrastjenkins/VulnerabilityTrendStep.java:183]: Why did another option get added to the pipeline?

Comment (serge17) [reply src/main/java/com/aspectsecurity/contrast/contrastjenkins/VulnerabilityTrendStep.java:183]: We had this option in the previous plugin version, so I added it for better compatibility.

File: src/main/java/com/aspectsecurity/contrast/contrastjenkins/VulnerabilityTrendHelper.java (hunk 1 above)
Comment (serge17) [reply src/main/java/com/aspectsecurity/contrast/contrastjenkins/VulnerabilityTrendHelper.java:252]: We don't use it anymore. We use 2 other similar methods.

**Comment by gmoore** [comment]
@serge17 Make sure this works with existing installs and get a release out (2.9)

**Pull request merged**